Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityControllerMappingProfile.cs

```diff
@@ -19,24 +19,25 @@ namespace Microsoft.MixedReality.Toolkit.Core.Definitions.Devices
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Controller Mapping Profile", fileName = "MixedRealityControllerMappingProfile", order = (int)CreateProfileMenuItemIndices.ControllerMapping)]
     public class MixedRealityControllerMappingProfile : BaseMixedRealityProfile
     {
-        private static Type[] controllerMappingTypes;
-
-        public static Type[] ControllerMappingTypes { get { CollectControllerTypes(); return controllerMappingTypes; } }
-
-        public static Type[] CustomControllerMappingTypes { get => (from type in ControllerMappingTypes where UsesCustomInteractionMapping(type) select type).ToArray(); }
-
         [SerializeField]
         [Tooltip("The list of controller templates your application can use.")]
         private MixedRealityControllerMapping[] mixedRealityControllerMappingProfiles = new MixedRealityControllerMapping[0];
 
         public MixedRealityControllerMapping[] MixedRealityControllerMappingProfiles => mixedRealityControllerMappingProfiles;
 
+#if UNITY_EDITOR
```

Comment:
Have we confirmed a .NET build works and deploys without this section in the build on both Windows and Standalone @davidkline-ms ?

Reply:
I have confirmed an il2pp build deploys and runs. there are other parts (interactables) of the project that are preventing successful .net backend builds.